src/parser/DefaultParser.js

```diff
@@ -24,6 +26,27 @@ class DefaultParser {
     this.pluginData.parseErrors = [];
   }
 
+  /**
+   * Set configuration into Components.
+   * @param {FileInput} configFile - File input to retrieve configuration.
+   */
+  parseConfiguration(configFile) {
+    const config = JSON.parse(configFile.content);
+
+    if (!config[this.pluginData.name]) {
+      return;
+    }
+
+    Object.keys(config[this.pluginData.name]).forEach((id) => {
+      const component = this.pluginData.getComponentById(id);
+
+      if (!component) {
+        return;
+      }
+      component.drawOption = new ComponentDrawOption(config[this.pluginData.name][id]);
```

Comment:
Consider not setting component.drawOption if it was not explicitely set in the config file as this could interfere with the layout generation

Reply:
good point :)
In old version we need to have all DrawOption instanciate, but not in the newest.